yamcs-core/src/main/java/org/yamcs/tctm/cfs/CfsEventDecoder.java

```diff
@@ -61,14 +63,19 @@ public Spec getSpec() {
         spec.addOption("msgIds", OptionType.LIST).withElementType(OptionType.INTEGER);
         spec.addOption("byteOrder", OptionType.STRING);
         spec.addOption("charset", OptionType.STRING);
-
+        spec.addOption("appNameMax", OptionType.INTEGER);
+        spec.addOption("eventMsgMax", OptionType.INTEGER);
```

Comment:
```suggestion
        spec.addOption("appNameMax", OptionType.INTEGER).withDefault(20);
        spec.addOption("eventMsgMax", OptionType.INTEGER).withDefault(122);
```

Reply:
Before I push anything, I'd like to point out that, for some reason, 53e7d8cd84be5eb4993b28a1ed07cfe6a874d90f breaks the yamcs webapp.

Here is a screenshot of what it shows:


![Screenshot from 2021-01-18 19-30-43](https://user-images.githubusercontent.com/68551873/104976944-fc38ae80-59c3-11eb-87bf-a2c1385cd42e.png)

The webapp is blank as opposed to showing the layout.

Anything before the commit goes back to the way it was.

Reply:
please recompile the yamcs-web after that commit. We have changed it to use the more generic file-transfer instead of cfdp so it needs recompilation. To do that (see README at the top of yamcs):

    cd yamcs-web/src/main/webapp
    npm install
    npm run build

Reply:
Tried again inside an isolated docker container  running `Ubuntu 18.04.5 LTS` and still have the same exact problem. I was on commit 24e683e2c8978b76019c3dcb704343a4841d7158 and recompiled everything; webapp and yamcs server.

Here is a screenshot of the issue:
![Screenshot from 2021-01-19 10-25-20](https://user-images.githubusercontent.com/68551873/105062904-bd950980-5a40-11eb-87fe-fb42675d1a87.png)

Reply:
Can you check in a standard yamcs master ./run-example.sh cfdp that it works?

There must be something specific to your configuration.

Reply:
Will do. But do note that yamcs works with my current configuration on commit f43083bb2539905260d10f397e8af633a3ba6218 or older. Will report feedback in a few minutes.

Reply:
`./run-example.sh cfdp` runs fine on `yamcs:master`

Reply:
when merged https://github.com/yamcs/yamcs/pull/491 should fix this recent issue

Reply:
ok, I merged the other 2 PRs so Lorenzo, please give it a try and we can merge this one also if everything is ok.